packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeHarpAndStringEach.yml

```diff
@@ -1,7 +1,7 @@
 languageId: typescript
 command:
   version: 1
-  spokenForm: take harp and string each
+  spokenForm: take harp and parse tree string each
```

Comment:
Note that we now distinguish between the "string" that's actually speakable and "parse tree string". These spoken forms are not tested, because there's no way to actually speak it, but I think it makes the test spoken form less confusing